flocker/provision/_aws.py

```diff
@@ -67,7 +79,13 @@ def _wait_until_running(instance):
                 instance_state=instance.state,
             ):
                 sleep(1)
-            instance.update()
+            try:
+                instance.update()
+            except EC2ResponseError as e:
+                _check_response_error(
+                    e,
+                    u"flocker:provision:aws:wait_until_running:retry"
+                )
```

Comment:
JP, you said likewise, but I am not sure likewise to what :-)
In any case, I''ll try `flocker.common.poll_until`